### PR TITLE
[FLINK-25800][docs] Update wrong links in the datastream/execution_mode.md page.

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

Update wrong links in the datastream/execution_mode.md page.

## Brief change log

Update wrong links in the datastream/execution_mode.md page.

## Verifying this change

Update wrong links in the datastream/execution_mode.md page.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
@flinkbot run azure

**Comment by RocMarshal** [comment]
Hi, @MartijnVisser Would you mind helping me to check it ? Thanks a lot.

File: docs/content.zh/docs/dev/datastream/execution_mode.md
Comment (wuchong) [inline docs/content.zh/docs/dev/datastream/execution_mode.md:204]: Why remove the `sum()` link?

Comment (RocMarshal) [reply docs/content.zh/docs/dev/datastream/execution_mode.md:204]: Glad to get your review .  @wuchong 

There isn't  any aggregate function section title in the docs/dev/datastream/operators/overview page, but we could make it mapping to the page   [flink/docs/content.zh/docs/dev/datastream/operators/windows.md](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/operators/windows/#aggregatefunction') .  

Please let me know what's your opinion.

**Comment by RocMarshal** [comment]
Hi, @MartijnVisser Could you help me to check it ? Thanks a lot.

File: docs/content.zh/docs/dev/datastream/execution_mode.md
Comment (MartijnVisser) [inline docs/content.zh/docs/dev/datastream/execution_mode.md:173]: The English version of the documentation currently on master https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/execution_mode/#event-time--watermarks still points to the `docs/dev/datastream/event-time/generating_watermarks` page. I think the current situation is the right page to point users to and therefore we shouldn't make this change.

File: docs/content.zh/docs/dev/datastream/execution_mode.md
Comment (MartijnVisser) [reply docs/content.zh/docs/dev/datastream/execution_mode.md:204]: The English documentation doesn't have a link to `sum()` so I think it's fine to remove the link here

File: docs/content.zh/docs/dev/datastream/execution_mode.md
Comment (MartijnVisser) [inline docs/content.zh/docs/dev/datastream/execution_mode.md:208]: Good catch 👍

File: docs/content.zh/docs/dev/datastream/operators/overview.md
Comment (MartijnVisser) [inline docs/content.zh/docs/dev/datastream/operators/overview.md:684]: This is now missing the last pieces which are on https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/operators/overview/#name-and-description

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
Thanks for the PR @RocMarshal - I do think that there are some changes which still need to be fixed before we can merge this

File: docs/content.zh/docs/dev/datastream/operators/overview.md
Comment (RocMarshal) [reply docs/content.zh/docs/dev/datastream/operators/overview.md:684]: @MartijnVisser Yes. I  executed the 'git rebase master'  command to resolve it.

**Review by RocMarshal — COMMENTED** [review]
@MartijnVisser @wuchong Thanks for your review. 
I made the change based on your opinions. 
PTAL. Thank you very much.

**Review by MartijnVisser — APPROVED** [review]
@RocMarshal I've rebased and squashed commits, I'll merge it when the CI has passed.

**Comment by MartijnVisser** [comment]
@RocMarshal Thanks again for the PR. Can you also create backports to the release-1.15 and release-1.14 branches?

**Comment by RocMarshal** [comment]
> @RocMarshal Thanks again for the PR. Can you also create backports to the release-1.15 and release-1.14 branches?

@MartijnVisser Thanks for the operation. I'm glad to do it and I'll create it asap.

**Comment by RocMarshal** [comment]
@MartijnVisser I created [BP-1.14](https://github.com/apache/flink/pull/19090) & [BP-1.15](https://github.com/apache/flink/pull/19091). 
PTAL. Thank you very much.

**Pull request merged**
